beacon-chain/state/stateutil/field_root_validator.go

```diff
@@ -15,14 +15,18 @@ import (
 )
 
 const (
-	// number of field roots for the validator object.
-	validatorFieldRoots = 8
+	// number of field roots for the validator object. ( validatorFields + validatorPaddingFields)
+	validatorFieldRoots = 16
```

Comment:
Tiny suggestion here.

```suggestion
	validatorFieldRoots = validatorFields + validatorPaddingFields
```

IDE will automatically display this number as 16. If then, following assertion is redundant.

```go
assert.Equal(t, validatorFieldRoots, validatorFields+validatorPaddingFields)
```